### ISSUE TITLE
Safer custom CSS implementation

### ISSUE THREAD

**Issue by wedebe** [E2OpenPlugins/e2openplugin-OpenWebif#1495]
https://github.com/E2OpenPlugins/e2openplugin-OpenWebif/commit/8e82cd5e71f184acf7173ebf9211fac4fd9d52b4
introduced custom CSS by OS-level file loading and processing.

A preferable solution from a security and simplicity standpoint would be to set a symbolic link of something like
`/usr/lib/enigma2/python/Plugins/Extensions/OpenWebif/plugin/public/css/owif-classic.css` -> `/etc/enigma2/owif-classic.css`
and
`/usr/lib/enigma2/python/Plugins/Extensions/OpenWebif/plugin/public/modern/css/owif-modern.css` -> `/etc/enigma2/owif-modern.css`

`<link rel="stylesheet" type="text/css" href="/css/owif-classic.css">`
and
`<link rel="stylesheet" type="text/css" href="/modern/css/owif-modern.css">`

thereby delegating file handling entirely to the browser which will merely receive a 404 error if no such file exists

**Comment by jbleyel** [comment]
My implementation prevents any errors.
The File is optional.

You can do this if you want.

```
#if $USERCSSCLASSIC != ''
<link rel="stylesheet" type="text/css" href="/css/owif-classic.css"></style>
#end if

```

USERCSSCLASSIC must be the info if file exists.

**Comment by jbleyel** [comment]
Please note! The css must be the last.

**Comment by jbleyel** [comment]
Please test this:

https://github.com/E2OpenPlugins/e2openplugin-OpenWebif/commit/c15ff7b05aadeb542c1cfd1f3227960f546932f3

**Comment by wedebe** [comment]
> Please test this:
> 
> [c15ff7b](https://github.com/E2OpenPlugins/e2openplugin-OpenWebif/commit/c15ff7b05aadeb542c1cfd1f3227960f546932f3)

Will do, thanks!

**Comment by wedebe** [comment]
@jbleyel , I get `ImportError: cannot import name 'symlink' from 'posixpath' (/usr/lib/python3.9/posixpath.pyc)`
even when manually running `>>> from posixpath import symlink`

**Comment by jbleyel** [comment]
Please try again.
https://github.com/E2OpenPlugins/e2openplugin-OpenWebif/commit/28227c690be0ed9954c2fa3dae38946fe83be0a4

**Comment by jbleyel** [comment]
fixed in 2.0
